src/org/opendatakit/common/cli/Operation.java

```diff
@@ -0,0 +1,60 @@
+package org.opendatakit.common.cli;
+
+import static java.util.Collections.emptySet;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Consumer;
+
+public class Operation {
```

Comment:
Stepping through the code in the debugger (iterating over a Set of Operations), I’m thinking a `toString` method might be useful. Not a high priority.